browser-test/src/landing_page.test.ts

```diff
@@ -1,9 +1,10 @@
-import {createTestContext} from './support'
+import {createTestContext, validateScreenshot} from './support'
 
 describe('the landing page', () => {
   const ctx = createTestContext()
 
   it('it has login options', async () => {
     expect(await ctx.page.textContent('html')).toContain('Continue as guest')
+    await validateScreenshot(ctx.page, 'landing_page')
```

Comment:
For consistency use "-" between words in screenshot name. Also we have validation in validateScreenshot for that. I'm not sure why it didn't fail for you.

Reply:
The regex only was testing for presence of those characters, not that the entire string matched it.  Fixed the regex and the name

Reply:
Ugh, thanks. I thought I tested it when this check was added originally. But probably not well enough.